src/rosdiscover/cli.py

```diff
@@ -129,6 +147,20 @@ def main() -> None:
     p.add_argument('config', type=argparse.FileType('r'), help=CONFIG_HELP)
     p.set_defaults(func=generate_acme)
 
+    p = subparsers.add_parser('observe',
+                              help='observes a robot running in a container and produces an '
+                                   'architecture',
+                              formatter_class=MultiLineFormatter)
+    p.add_argument('--container', type=str, help='The container where the ROS system is running')
```

Comment:
I think this should be a mandatory, positional argument?

Reply:
Done.